lib/android.js

```diff
@@ -834,7 +833,7 @@ function findSDK(dir, config, androidPackageJson, callback) {
 			return callback(true);
 		}
 
-		var file = path.join(dir, 'tools', 'source.properties');
+		var file = path.join(dir, 'platform-tools', 'source.properties');
```

Comment:
I wonder if we could just remove this check altogether (or make it no longer required) as it seems they've phased this folder out?

Reply:
I didn't test that, just wanted to make minimal changes first so it will work :smile:

Reply:
I removed one of the checks since it was checking the plattform-tools folder twice